test/integration/application-log-assersion/karma-reporter/index.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+var _ = require("lodash");
+
+var formatter = require("./formatter");
+var writer = require("./writer");
+
+var CucumberReporter = function (baseReporterDecorator, config, logger, helper) {
```

Comment:
I actually expected that you can use an existing reporter for this, e.g. https://www.npmjs.com/package/karma-cucumber-js